src/Client/Style.fs

```diff
@@ -38,10 +37,10 @@ let buttonLink cssClass onClick elements =
         OnTouchStart (fun _ -> onClick())
         Style [ unbox("cursor", "pointer") ] ] elements
 
-let internal onEnter msg dispatch =
-    function
-    | (ev:React.KeyboardEvent) when ev.keyCode = 13. ->
-        ev.preventDefault()
-        dispatch msg
-    | _ -> ()
-    |> OnKeyDown
+let onEnter msg dispatch =
+    OnKeyDown (fun (ev:React.KeyboardEvent) ->
```

Comment:
I don't like the use of "function" and "|>" and so on in this, it's too whack for the beginner.